crates/apollo-parser/src/parser/grammar/ty.rs

```diff
@@ -85,6 +87,7 @@ fn parse<'a>(p: &mut Parser<'a>) -> Result<(), Token<'a>> {
 /// *NamedType*:
 ///     Name
 pub(crate) fn named_type(p: &mut Parser) {
+    // TODO(@goto-bus-stop) can we make this error instead if no name is found?
```

Comment:
Yes, I don’t know if the parser as a whole has inputs where this makes an observable difference, but this function silently not doing anything seems bad on principle. I’ve filed https://github.com/apollographql/apollo-rs/issues/829 so we don’t forget to look into it.